docs/userguide/main.md

```diff
@@ -23,16 +23,30 @@ spec:
     serviceName: my-first-cluster
   dashboards:
     enable: true
+    replicas: 2
```

Comment:
As this Cluster definition is supposed to be a minimal cluster for demo purposes this should be only 1 replica.

Reply:
Noted and pushed the change to default as `replicas: 1`.